src/entities/flier.rs

```diff
@@ -0,0 +1,80 @@
+use amethyst::{
+    assets::{Handle, Prefab},
+    core::{
+        math::{Vector2, Vector3},
+        Transform, WithNamed,
+    },
+    ecs::prelude::World,
+    prelude::{Builder, WorldExt},
+    renderer::{palette::Srgba, resources::Tint, transparent::Transparent},
+};
+
+use crate::{
+    components::{
+        Animation, AnimationId, AnimationPrefabData, Boundary, Collidee, Collider, Direction,
+        Directions, Flier, GenericBox, Motion,
+    },
+    resources::Context,
+};
+
+pub fn load_flier(world: &mut World, prefab: Handle<Prefab<AnimationPrefabData>>, ctx: &Context) {
+    // TODO: There is a lot of padding on each sprite frame.
+    // Should the collision/bounding boxes be reshaped accordingly?
```

Comment:
Yep, I can clean it up. I was tweaking some of the values but was struggling to tell the difference. I meant to check Amethyst's docs instead of just hacking and hoping.

Reply:
I think it looks much better now. Let me know what you think

Reply:
Perfect!